src/network/netdev/tunnel.c

```diff
@@ -745,7 +749,9 @@ static int netdev_tunnel_verify(NetDev *netdev, const char *filename) {
                 return log_netdev_error_errno(netdev, SYNTHETIC_ERRNO(EINVAL),
                                               "The local address cannot be '%s' when Independent= or AssignToLoopback= is enabled, ignoring.",
                                               strna(netdev_local_address_type_to_string(t->local_type)));
-
+        if (t->ignore_df && t->pmtudisc)
+                return log_netdev_error_errno(netdev, SYNTHETIC_ERRNO(EINVAL),
+                                              "IgnoreDF cannot be enabled when DiscoverPathMTU= is enabled, ignoring.");
```

Comment:
Strike ", ignoring.".

All the other messages in this function are wrong too: when we return an error, we shouldn't say "ignoring".
It'd be nice if you fixed that (in a separate commit message).

Reply:
done